src/System.Device.Gpio/System/Device/Media/VideoConnectionSettings.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Media
+{
+    /// <summary>
+    /// The connection settings of a video device.
+    /// </summary>
+    public class VideoConnectionSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="VideoConnectionSettings"/> class.
+        /// </summary>
+        /// <param name="busId">The bus ID the video device is connected to.</param>
+        public VideoConnectionSettings(int busId)
+        {
+            BusId = busId;
+        }
+
+        /// <summary>
+        /// The bus ID the video device is connected to.
+        /// </summary>
+        public int BusId { get; }
+
+        /// <summary>
+        /// The size of video device captured image.
+        /// </summary>
+        public (uint Width, uint Height) CaptureSize { get; set; } = (0, 0);
+
+        /// <summary>
+        /// The pixel format of video device captured image.
+        /// </summary>
+        public PixelFormat PixelFormat { get; set; } = PixelFormat.YUYV;
+
+        /// <summary>
+        /// The exposure type of video device.
+        /// </summary>
+        public ExposureType ExposureType { get; set; }
```

Comment:
These settings I refer to the settings provided by raspistill. These are commonly used settings. V4L2 can set too many settings😅.